xla/service/gpu/transforms/command_buffer_scheduling_test.cc

```diff
@@ -1249,32 +1184,14 @@ TEST_F(CommandBufferSchedulingTest, DynamicSliceFusionStaticSlicing) {
     ASSERT_NE(child->kind(), Thunk::kCommandBuffer);
   }
 
-  // FUSION off, COLLECTIVES on -> command buffer because static slices.
```

Comment:
This test has become a little unreadable now, I'm not sure I understand what's being tested. Some cases have disappeared, and others (e.g. `FUSION on, COLLECTIVES on` above) no longer match the comments.

Can we restructure this to follow a structured pattern?

If you want to set default options for the whole test, it's OK to subclass `CommandBufferSchedulingTest` to enable a few more debug options. That might declutter the test a little as well.